app/src/debug/java/com/teamagam/gimelgimel/app/common/utils/Constants.java

```diff
@@ -2,7 +2,7 @@
 
 
 public class Constants {
-    public static final float LOCATE_ME_BUTTON_ALTITUDE_METERS = 500;
+    public static final float LOCATE_ME_BUTTON_RESOLUTION_LEVEL = 1;
```

Comment:
Yes.
Circle CI builds the application in release mode. If it wont find his constant it'll fail.